### DISCUSSION TITLE
add wwtp_structure to main ws view

### DISCUSSION THREAD

**Discussion by cymed** [teksi/wastewater#38] [Show and tell]
VSA DSS 2020 has a new wastewater structure type tww_od.wwtp_structure that is not included in the main view.

**Comment by sjib** [comment]
There are more new subclasses in VSA-DSS 2020 that we have not yet included:

- Abflusslose Toilette
- KLARA
- ARABauwerk (wwtp_structure)

I think it would rather make sense to add new seperate views (and forms) for those than include them to the main view, as they have usually no cover and other attributes than manhole, special_structure, infiltration_installation and discharge_point.
I could also imagine, that these classes are maintained by other users than those who maintain the rest of the sewer network.

**Reply by cymed** [reply]
Would be ok for me to group these three. Ideally based on the current ws python script.

**Comment by sjib** [comment]
Implementation see https://github.com/teksi/wastewater/pull/42
